app/_src/gateway/plugin-development/get-started/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Introduction 
+book: plugin_dev_getstarted
+chapter: 1
+---
+
+This tutorial will walk you through all the steps to build a new custom plugin for {{site.base_gateway}}.
+The plugin you build will transform HTTP traffic as it passes through the API gateway, and give you
+a solid starting point for building custom plugins to solve your specific business needs.
+
+The guide starts by setting up a new plugin project, showing how folders are structured and explaining 
+code file contents and best practices. The guide proceeds by describing testing tools and how to 
+build automated tests for your custom plugins. The guide continues by showing how to add 
+plugin configuration to the project allowing for runtime behavior changes.
+
+Later chapters of the guide go further to provide guidance on advanced topics including 
+consuming external services from within your plugin and plugin deployments.
+
+Use the links on the bottom of these pages to navigate the guide. If you experience any issues
+with this sample code or guidance in these pages, please reach out by 
+[reporting an issue on GitHub](https://github.com/Kong/docs.konghq.com/issues).
+
+{:.note}
+> **Note**: While not required, an understanding of the [Lua](https://www.lua.org/about.html) language
+> is helpful for this guide.
+
+{:.note}
+> **Note**: This guide is written to help you quickly build a custom plugin 
+> from the beginning. For advanced topics on plugin development with {{site.base_gateway}},
+> be sure to see the remaining areas of documentation in the 
+> [Plugin Development Section](/gateway/{{page.release_version}}/plugin-development/).
```

Comment:
Just try to add warning to users, only using PDK.

```suggestion
> [Plugin Development Section](/gateway/{{page.release_version}}/plugin-development/).

{:.note}
> **Note**: {{site.base_gateway}} ships [Plugin Development Kit](/gateway/{{page.release}}/plugin-development/pdk/) to simplify custom plugin development. Please avoid using any other unpublished {{site.base_gateway}} modules as they vary frequently and may break custom plugins.
```

Reply:
I'd like to find another place for this as I think it's a bit out of context in this introduction page.   I'm going to move it to this section here where we first introduce the PDK.

https://deploy-preview-7068--kongdocs.netlify.app/gateway/unreleased/plugin-development/get-started/setup/#add-handler-logic

Reply:
@outsinre I'm not totally clear on what we mean by "other unpublished Kong Gateway modules", so I think readers will be too.  What do we mean by unpublished Kong Gateway modules?

Reply:
@rspurgeon I mean the whole Kong Gateway code except the PDK part.

Our PDK interface is consistent but other code changes daily. If customer copy that code to the custom plugin, it may break in incoming releases.

Reply:
@outsinre Added a note here to address the concern.  LMK if this works for you. https://github.com/Kong/docs.konghq.com/pull/7068/commits/315437cc2807654ae0af8e68968dd329a4c0174f